kit/Kit.cpp

```diff
@@ -1799,6 +1803,21 @@ class Document final : public DocumentManagerInterface
     bool _inputProcessingEnabled;
 };
 
+#if !defined FUZZER && !defined BUILDING_TESTS
+
+// When building the fuzzer we link LOOLWSD.cpp into the same executable so the
+// Protected::addOneRecording() there gets used. When building the unit tests the one in
+// ProfileZone.cpp gets used.
+
+void ProfileZone::addOneRecording(const std::string &sRecording)
+{
+    if (!config::getBool("trace_event[@enable]", false))
```

Comment:
I'd expect this to be heinously slow too - for every line we add as we trace; can we have a static / global variable that is set at load time (?)

Reply:
Ask @Ashod , who explicitly said I should use that instead of a global variable.

Reply:
You use a global variables. The expert say no, we have an API for that. You use that API. Another expert says no, use a global variable. Repeat.

Reply:
From Mattermost: "@tml: fyi, the kit now has full access to loolwsd.xml :slightly_smiling_face: so we don't need to pass any new configs to kit explicitly and we don't need any new global or other variables in kit... see ConfigUtil.{hpp,cpp}."

Reply:
heh - so a locally scoped static bool = would make both happy I think.